polyfill/test/PlainDate/prototype/toPlainDateTime/time-undefined.js

```diff
@@ -3,23 +3,14 @@
 
 /*---
 esid: sec-temporal.plaindate.prototype.toplaindatetime
+includes: [temporalHelpers.js]
 features: [Temporal]
 ---*/
 
 const date = new Temporal.PlainDate(2000, 5, 2);
 
 const explicit = date.toPlainDateTime(undefined);
-assert.sameValue(explicit.hour, 0, "default time is midnight");
-assert.sameValue(explicit.minute, 0, "default time is midnight");
-assert.sameValue(explicit.second, 0, "default time is midnight");
-assert.sameValue(explicit.millisecond, 0, "default time is midnight");
-assert.sameValue(explicit.microsecond, 0, "default time is midnight");
-assert.sameValue(explicit.nanosecond, 0, "default time is midnight");
+TemporalHelpers.assertPlainDateTime(explicit, 2000, 5, "M05", 2, 0, 0, 0, 0, 0, 0, "explicit");
```

Comment:
Can we put something like "default time is midnight - explicit"? (And same below)